ocs_ci/ocs/ui/block_pool.py

```diff
@@ -64,7 +64,7 @@ def check_pool_existence(self, pool_name):
             bool: True if pool is in the list of pools page, otherwise False
 
         """
-        self.navigate_cluster_overview_page()
+        # self.navigate_cluster_overview_page()
```

Comment:
why not remove this line?

Reply:
removed all in the lastest commit, I forgot to push that